### PR TITLE
Add API Capabilities and Meeting Topics/Summary

### PR DESCRIPTION
**Pull request by sandersms**

### CODE REVIEW

**Comment by sandersms** [comment]
Includes the Capabilities desired view from Toshi Kani at HPC and the Application API Requirements inputs from John McDowall at Palo Alto Networks.  This is a starting point view of the Capabilities that need to be considered as part of the API definitions.

File: CAPABILITIES.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Desired (Must Have) Capabilities
+
+- Define a common northbound APIs between client controllers (e.g., Provisioner/Orchestrator, K8s CNI/CSI, Ansible/Terraform, etc.) and DPU, which allows the client controllers to support multi-vendor DPUs with no/minimal change
+- Provide the following high level features:
+  - Network
+    - Provision VPC
+      - VLAN, VXLAN, BGP/EVPN, ...
+    - Provision vNIC
+      - Static or Dynamic vNICs, SR-IOV, S-IPV, IPAM, Routing, ...
```
[the remark this answers is not in the document]
Comment (sandersms) [reply CAPABILITIES.md:11]: Thanks - will fix that.

**Review by toshikani — APPROVED** [review]

File: CAPABILITIES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sandersms) [reply CAPABILITIES.md]: Corrected typo

**Review by glimchb — APPROVED** [review]
lgtm 
@artek-koltun ?

**Pull request merged**